### DISCUSSION TITLE
ABMS x SD

### DISCUSSION THREAD

**Discussion by lufm** [gama-platform/gama#47] [General]
Hi everyone! Happy Friday! 

I´ve been reflecting on the approach I use in my research:  ABMS or Systems Dynamics One important criterion might be the tool I would deploy: GAMA or ithink/Powersim/Vensim. I would like to know if anyone here has ever modeled and simulated the same case both in GAMA and other SD software and if so what was the outcome? How would you compare both tools?  

Many thanks! Merci!  

Luciana

**Comment by benoitgaudou** [comment]
Dear,
First of all, you can notice you can do both in GAMA (agent-based model and
System Dynamics (using equation)).
After that it should exist in the GAMA model library an example of an
apidémiological model modeled as an SD and using agents.
Resultats are quite similar but ABM has a lot more fluctuations.

Cheers

Benoit

**Reply by lufm** [reply]
Hi Benoit!  Thanks a lot for the reply! It´s really impressive how complete the Gama Platform is. Still I wonder if I could access any comparison of outcomes using GAMA and one of the softwares iThink/Powersim/Vensin. 
Cheers,
Luciana
